brd-ios/breadwallet/src/Helpers/PickerViewViewController.swift

```diff
@@ -146,7 +146,14 @@ extension PickerViewViewController: UIPickerViewDataSource, UIPickerViewDelegate
     }
     
     public func pickerView(_ pickerView: UIPickerView, didSelectRow row: Int, inComponent component: Int) {
```

Comment:
what does this change do actually?
makes em go in circles? 🤔

Reply:
each time you pick, it gave you also the component it changed. i don't need the component, i need to row inside the component. it does not go in circles.